src/CMakeLists.txt

```
@@ -275,6 +275,10 @@ if(LLVM)
   execute_process(COMMAND ${LLVM_CONFIG}  --cxxflags OUTPUT_VARIABLE LLVM_CONFIG_CXXFLAGS OUTPUT_STRIP_TRAILING_WHITESPACE)
   execute_process(COMMAND ${LLVM_CONFIG}  --system-libs OUTPUT_VARIABLE LLVM_CONFIG_SYSTEM_LIBS OUTPUT_STRIP_TRAILING_WHITESPACE)
   message(STATUS "llvm-config: libdir '${LLVM_CONFIG_LIBDIR}' | ldflags '${LLVM_CONFIG_LDFLAGS}' | libs '${LLVM_CONFIG_LIBS}' | system libs '${LLVM_CONFIG_SYSTEM_LIBS}' | cxxflags: ${LLVM_CONFIG_CXXFLAGS} | includedir: ${LLVM_CONFIG_INCLUDEDIR}")
+
+  # LEANC_CC is "cc" by default which could point to gcc.
+  # However the LLVM backend needs to use leanc on bitcode files -> it needs to be clang
+  set(LEANC_CC "clang")
```

```suggestion
  set(LEANC_CC ${CMAKE_CXX_COMPILER})
```
We might want to reuse the CXX compiler that we picked up from CMake config, rather than use `clang` that could point to a _different_ compiler.

---

should've been CMAKE_C_COMPILER :wink: